services/system/Accounts/ui/src/hooks/useCreateAccount.ts

```diff
@@ -39,18 +37,6 @@ export const useCreateAccount = (token: string) => {
 
       await importAccount(account);
 
-      void (await supervisor.functionCall({
-        method: "loginDirect",
-        params: [
-          {
-            app: inviteToken.app,
-            origin: inviteToken.appDomain,
-          },
-          account,
-        ],
-        service: "accounts",
-        intf: "admin",
-      }));
     },
```

Comment:
The idea was that the user is choosing to accept the invite with a new account, as opposed to one of their existing ones.